docs/source/quick_start/first_run.rst

```diff
@@ -0,0 +1,163 @@
+
+First Run
+=========
+
+Now, you have installed MCDR, it's time to run it for the first time
+
+.. note::
+
+    The most common use of MCDR is to control a Minecraft server, so we also take this as the demonstrated use case here
+
+.. tip::
+
+    To manage a Minecraft server with MCDR, it is recommended to have some basic understanding of common Minecraft servers
+    
+    Which means, you should know:
+
+    * The Minecraft servers: what they are, how to configure (one of) them
+    * Start command of Minecraft server: what it is, what its arguments mean
+    * Console of the server: what it prints, what command can you send to it
+    * How to manage the server: configure, mods / plugins etc.
+    
+    If you have not, Google is your friend
+
+Prepare
+-------
+
+To manage a Minecraft server with MCDR, you should have a **well-configured** Minecraft server
+
+For example, this is the typical directory structure of a vanilla Minecraft server:
+
+.. code-block:: text
+
+    ./
+    ├── libraries/
+    ├── logs/
+    ├── versions/
+    ├── world/
+    ├── banned-ips.json
+    ├── banned-players.json
+    ├── eula.txt
+    ├── ops.json
+    ├── minecraft_server.jar
+    ├── server.properties
+    ├── usercache.json
+    └── whitelist.json
+
+With this command to start it:
+
+.. code-block:: bash
+
+    java -Xms1G -Xmx2G -jar minecraft_server.jar nogui
+
+You can also use other servers, such as Fabric, Spigot, Paper, etc.
+
+.. tip::
+
+    Configure and troubleshoot your server **BEFORE** you introduce MCDR. Otherwise, you can't distinguish whether a problem is caused by MCDR or your server
+
+    MCDR was born as a daemon for common servers of Minecraft: Java Edition. Most plugins are predicated on that. If you want to run MCDR with a Bedrock Edition server, consider using a Java Edition server with `GeyserMC <https://geysermc.org/>`__ to provide Bedrock Edition compatibility
```

Comment:
JE + Geyser is not a workaround if user does want a BE server

I think you can just remove "If you want to run MCDR with a Bedrock Edition server, consider using a Java Edition server with `GeyserMC <https://geysermc.org/>`__ to provide Bedrock Edition compatibility" since it does not completely resolve the problem, and we won't need to update it if a new native solution is found in the future

Reply:
sounds fair